### PR TITLE
Add iconUrl to custom token integration example, and other improvements

### PR DESCRIPTION
**Pull request by fragosti**
Figured this would be useful since we don't document it anywhere.

### CODE REVIEW

File: instant/Questions-About-Instant.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ To prevent massive price ranges, 0x Instant calculates a maximum amount of asset
 
 #### Q: What is assetMetaData?
 
-In order to provide a good user experience, Instant requires data that is not available on-chain. This includes the decimals, symbol and name of the token along with optional information like primaryColor that informs Instant how to theme itself when that token is selected. It also requires to know whether the token is ERC20 or some other standard via the assetProxyId field. At the moment, only ERC20 is supported and the value for assetProxyId should always be “0xf47261b0”.
+In order to provide a good user experience, Instant requires data that is not available on-chain. This includes the decimals, symbol and name of the token along with optional information like primaryColor that informs Instant how to theme itself when that token is selected. It also requires to know whether the token is ERC20 or some other standard via the assetProxyId field. At the moment, only ERC20 is supported and the value for assetProxyId should always be “0xf47261b0”. Check out the 0x v2 protocol specification for more information on [assetData](https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md#assetdata).
```
Comment (steveklebanoff) [inline instant/Questions-About-Instant.md]: A few minor notes, not sure if they fall into scope or not:

- I feel like this is missing the description of the shape of the object and indicating the  the key of the object is assetData
- I think this should be grouped next to "What is assetData?", esp. with links to subsections not working, I first scrolled to "What is assetData" and mistook it for the "What is assetMetaData?" section.  Maybe these could even be combined...
- I feel like the "What is assetData?" section links out to a highly technical article description of assetData -- I think describing how to generate assetData in JS/TS or even linking out to a very simple codesandbox could go a long way towards helping people

File: instant/Get-Started-With-Instant.md [hunk 2]
```diff
@@ -150,15 +150,17 @@ zeroExInstant.render(
...
                 name: 'My Custom Token',
+                primaryColor: '#F2F7FF', // Optional
+                iconUrl: 'https://cdn.icons.com/my_icon.svg', // Optional
             },
         },
     },
     'body',
 );
 ```
+The icon referenced by `iconUrl` will go on top of a 26x26 circle that has `primaryColor` as a background. If an `iconUrl` is not provided, the specified token `symbol` ("XXX" in this case) will be displayed over the circle in white.
```
Comment (steveklebanoff) [inline instant/Get-Started-With-Instant.md]: Nice 👍

File: instant/Get-Started-With-Instant.md [hunk 3]
```diff
@@ -139,7 +139,7 @@ zeroExInstant.render(
 
 #### Providing a Custom Token
 
-Your token may not be currently supported by Instant by default. Check [here](https://github.com/0xProject/0x-monorepo/blob/development/packages/instant/src/data/asset_meta_data_map.ts) for a list of supported tokens. Check [here](https://0xproject.com/wiki#Questions-About-Instant) for more information about the object being passed in.
+Your token may not be currently supported by Instant by default. Check [here](https://github.com/0xProject/0x-monorepo/blob/development/packages/instant/src/data/asset_meta_data_map.ts) for a list of supported tokens. Check "What is assetMetaData?" in [the questions section](https://0xproject.com/wiki#Questions-About-Instant) for more information about the object being passed in.
```
Comment (steveklebanoff) [inline instant/Get-Started-With-Instant.md]: its verbose, but I feel like we need to specify: "for a list of supported tokens out of the box" or "for a list of tokens supported by default".  I might be overthinking it, but this helps indicate we can _support_ other tokens, just not by default

File: instant/Questions-About-Instant.md (hunk 1 above)
Comment (fragosti) [reply instant/Questions-About-Instant.md]: I broke down the fields but the code sandbox will have to be for another time :)

File: instant/Questions-About-Instant.md [hunk 4]
```diff
@@ -16,7 +16,63 @@ Check out this [article](https://0xproject.com/wiki#Web3-Provider-Explained) in
 
 #### Q: What is assetData?
 
-Check out the 0x v2 protocol specification for more information on [assetData](https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md#assetdata).
+As we now support multiple [token transfer proxies](https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md#assetproxy), the identifier of which proxy to use for the token transfer must be encoded, along with the token information. Each proxy in 0x v2 has a unique identifier. If you're using 0x.js there will be helper methods for this [encoding](https://0xproject.com/docs/0x.js#assetDataUtils-encodeAssetProxyId) and [decoding](https://0xproject.com/docs/0x.js#assetDataUtils-decodeAssetProxyId).
```
Comment (steveklebanoff) [inline instant/Questions-About-Instant.md]: The encoding link seems broken

File: instant/Questions-About-Instant.md [hunk 5]
```diff
@@ -16,7 +16,63 @@ Check out this [article](https://0xproject.com/wiki#Web3-Provider-Explained) in
 
 #### Q: What is assetData?
 
-Check out the 0x v2 protocol specification for more information on [assetData](https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md#assetdata).
+As we now support multiple [token transfer proxies](https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md#assetproxy), the identifier of which proxy to use for the token transfer must be encoded, along with the token information. Each proxy in 0x v2 has a unique identifier. If you're using 0x.js there will be helper methods for this [encoding](https://0xproject.com/docs/0x.js#assetDataUtils-encodeERC20AssetData) and [decoding](https://0xproject.com/docs/0x.js#assetDataUtils-decodeAssetProxyId).
```
Comment (steveklebanoff) [inline instant/Questions-About-Instant.md]: Nice 👍

**Comment by steveklebanoff** [comment]
The assetData section goes a little deeper and might be a little more intimidating than I would have expected, but it gives the user links to resources and to the JS/TS helper functions which I think are the most important things.  This looks good to me overall 👍   Thanks for putting up with the requests for changes ;) This is a big improvement over the existing docs.

**Pull request merged**